doc/classes/StringName.xml

```diff
@@ -33,12 +33,815 @@
+			<description>
+				Returns the SHA-256 hash of the string as an array of bytes.
+			</description>
+		</method>
+		<method name="sha256_text" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the SHA-256 hash of the string as a string.
+			</description>
+		</method>
+		<method name="similarity" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="text" type="String" />
+			<description>
+				Returns the similarity index ([url=https://en.wikipedia.org/wiki/S%C3%B8rensen%E2%80%93Dice_coefficient]Sorensen-Dice coefficient[/url]) of this string compared to another. A result of 1.0 means totally similar, while 0.0 means totally dissimilar.
+				[codeblock]
+				print("ABC123".similarity("ABC123")) # Prints "1"
+				print("ABC123".similarity("XYZ456")) # Prints "0"
+				print("ABC123".similarity("123ABC")) # Prints "0.8"
+				print("ABC123".similarity("abc123")) # Prints "0.4"
+				[/codeblock]
+			</description>
+		</method>
+		<method name="simplify_path" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns a simplified canonical path.
+			</description>
+		</method>
+		<method name="split" qualifiers="const">
+			<return type="PackedStringArray" />
+			<param index="0" name="delimiter" type="String" default="&quot;&quot;" />
+			<param index="1" name="allow_empty" type="bool" default="true" />
+			<param index="2" name="maxsplit" type="int" default="0" />
+			<description>
+				Splits the string by a [param delimiter] string and returns an array of the substrings. The [param delimiter] can be of any length. If [param delimiter] is an empty string, each substring will be a single character.
+				If [param allow_empty] is [code]true[/code], and there are two adjacent delimiters in the string, it will add an empty string to the array of substrings at this position.
+				If [param maxsplit] is specified, it defines the number of splits to do from the left up to [param maxsplit]. The default value of [code]0[/code] means that all items are split.
+				If you need only one element from the array at a specific index, [method get_slice] is a more performant option.
+				[b]Example:[/b]
+				[codeblocks]
+				[gdscript]
+				var some_string = "One,Two,Three,Four"
+				var some_array = some_string.split(",", true, 1)
+				print(some_array.size()) # Prints 2
+				print(some_array[0]) # Prints "Four"
+				print(some_array[1]) # Prints "Three,Two,One"
+				[/gdscript]
+				[csharp]
+				var someString = "One,Two,Three,Four";
+				var someArray = someString.Split(",", true); // This is as close as it gets to Godots API.
+				GD.Print(someArray[0]); // Prints "Four"
+				GD.Print(someArray[1]); // Prints "Three,Two,One"
+				[/csharp]
+				[/codeblocks]
+				If you need to split strings with more complex rules, use the [RegEx] class instead.
+			</description>
+		</method>
+		<method name="split_floats" qualifiers="const">
+			<return type="PackedFloat32Array" />
```

Comment:
This seems the line that fails one of the Linux builds. The line should be `<return type="PackedFloat64Array" />` instead.

Reply:
changing it to `PackedFloat64Array` would fail it on other platforms, (especially since its copied from `String.xml`), theres a weird bug going on here,

Reply:
i really cannot replicate this on WSL Ubuntu

Reply:
OH the documentation has changed this week